test/sqlsrv/sqlsrv_fetch_invalid_types.phpt

```diff
@@ -0,0 +1,142 @@
+}
```

Comment:
Is this function really needed here? Since it is just a one liner (although a very long time) and it's only being called in PopulateTestTable